yt_dlp/extractor/gedidigital.py

```diff
@@ -1,198 +1,104 @@
-                                'vbr': int_or_none(mobj.group(2)),
-                            })
-                        if not f.get('vbr'):
-                            f['vbr'] = int_or_none(self._search_regex(
-                                r'-video-rrtv-(\d+)', v, 'abr', default=None))
-                    formats.append(f)
-            elif t == 'param':
-                if n in ['image_full', 'image']:
-                    thumb = v
-                elif n == 'videoDuration':
-                    duration = int_or_none(v)
-
-        self._clean_formats(formats)
+        for stream in streams:
+            if isinstance(stream, str):
+                continue
+            if stream['type'] == 'video/mp4':
+                bitrate = self._search_regex(r'video-rrtv-(\d+)', stream['src'], 'vbr', None)
+                formats.append({
+                    'format_id': f'mp4-{bitrate}' if bitrate else 'mp4',
+                    'url': stream['src'],
+                    'ext': 'mp4',
+                    'vcodec': 'avc1',
+                    'acodec': 'mp4a',
+                    'vbr': int_or_none(bitrate),
+                })
+            elif stream['type'] == 'application/x-mpegURL':
+                new_formats = self._extract_m3u8_formats(stream['src'], video_id)
+                for fmt in new_formats:
+                    fmt.setdefault('vbr', int_or_none(self._search_regex(r'/hls/_(\d+)/', fmt['url'], 'vbr', None)))
+                formats.extend(new_formats)
+            elif stream['type'] == 'audio/mp3':
+                bitrate = self._search_regex(r'mp3-audio-(\d+)', stream['src'], 'vbr', None)
+                formats.append({
+                    'format_id': f'mp3-{bitrate}' if bitrate else 'mp3',
+                    'url': stream['src'],
+                    'ext': 'mp3',
+                    'acodec': 'mp3',
+                    'abr': int_or_none(bitrate),
+                    'tbr': int_or_none(bitrate),
+                })
+            else:
+                formats.append({
+                    'format_id': stream['type'],
+                    'url': stream['src'],
+                })
 
         return {
             'id': video_id,
-            'title': title,
-            'description': self._html_search_meta(
-                ['twitter:description', 'og:description', 'description'], webpage),
-            'thumbnail': thumb or self._og_search_thumbnail(webpage),
+            'display_id': slug,
+            'title': data.get('videoTitle') or self._html_search_meta(['twitter:title', 'og:title'], webpage),
+            'description': self._html_search_meta(['twitter:description', 'og:description', 'description'], webpage, default=None),
             'formats': formats,
-            'duration': duration,
+            'thumbnail': data.get('posterSrc') or self._og_search_thumbnail(webpage),
+            'uploader_id': data.get('accountId'),
+            'duration': int_or_none(data.get('videoLenght')),
```

Comment:
is `videoLenght` a typo?

Reply:
Probably it's a typo on their side, checked it multiple times and it's really `videoLenght`